mLRS/CommonTx/mlrs-tx.cpp

```diff
@@ -762,7 +778,28 @@ INITCONTROLLER_END
             connect_tmo_cnt--;
         }
 
-        leds.Tick_ms(connected());
+        DECc(tx_tick, SYSTICK_DELAY_MS(Config.frame_rate_ms));
+
+        if (tx_tick == 2) {
+            isInTimeGuard = true; // prevent extra work
+        }
+        if (tx_tick == 1) {
+            doPreTransmit = true; // trigger next cycle
+            pretransmit_tstamp_us = micros16();
+        }
+        if (tx_tick == 0) {
+//            crsf.TelemetryStart();
+//            whileTransmit.Trigger();
+        }
+
+        link_task_tick_ms();
+
+        if (!doPreTransmit) doSysTask2++;
+    }
+    if (doSysTask2) {
+        doSysTask2--;
```

Comment:
These 4 lines are a bit confusing to me as they appear to be an obscure way to say just "if (!doPreTransmit) {"
Since doSysTask2 is not referenced anywhere else that I can see.

Reply:
that's true, I found it slimmer though, didn't like the identing

Reply:
I'd much prefer the indenting!